django/db/models/expressions.py

```diff
@@ -170,8 +170,6 @@ class BaseExpression:
     # aggregate specific fields
     is_summary = False
     _output_field_resolved_to_none = False
-    # Can the expression be used in a WHERE clause?
-    filterable = True
```

Comment:
`filterable` is a documented API, so we should keep it untouched in this patch. We can deprecate it in a follow up PR.

Reply:
Makes sense, I'll keep it around!